src/averageAges.js

```diff
@@ -37,7 +51,26 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter(person => person.sex === 'f');
+
+  const countOfWomen = women.filter(isMother => {
```

Comment:
Replace the if statement with ternary operator

 ```suggestion
 const countOfWomen = women.filter(isMother =>
  withChildren
    ? people.some(person => person.mother === isMother.name)
    : people.some(person => isMother.name === person.name)
)
.reduce(count => count + 1, 0);
```